bitrise.yml

```diff
@@ -0,0 +1,31 @@
+---
+format_version: '6'
+default_step_lib_source: https://github.com/bitrise-io/bitrise-steplib.git
+project_type: ios
+trigger_map:
+- tag: "*.*.*"
+  workflow: post-release
+workflows:
+  post-release:
+    steps:
+    - set-env-var@0:
+        inputs:
+        - destination_keys: NEW_VERSION
+        - value: "$BITRISE_GIT_TAG"
+    - trigger-bitrise-workflow@0:
+        inputs:
+        - api_token: "$GLIA_IOS_PODSPECS_BUILD_TRIGGER_TOKEN"
+        - workflow_id: upload_core_sdk_podspec
+        - exported_environment_variable_names: NEW_VERSION
+        - app_slug: "$GLIA_IOS_PODSPECS_APP_SLUG"
+app:
+  envs:
+  - opts:
+      is_expand: false
+    BITRISE_PROJECT_PATH: Example/Example.xcworkspace
```

Comment:
True, neither envs are needed. I have removed them, which you can see [here](https://github.com/salemove/ios-bundle/pull/78/files#diff-61d1ee831cb1df242161a7d23171914a1d5d628c7659b52db957d80a9394bd68R21) because they are missing. Unfortunately I didn't make an additional fixup commit but fixed it in the same commit.